src/content/docs/en/guides/client-side-scripts.mdx

```diff
@@ -8,6 +8,32 @@ i18nReady: true
 
 You can add interactivity to your Astro components without [using a UI framework](/en/core-concepts/framework-components/) like React, Svelte, Vue, etc. using standard HTML `<script>` tags. This allows you to send JavaScript to run in the browser and add functionality to your Astro components.
 
+## Client-Side Scripts
+
+Astro components support adding client-side interactivity using standard HTML `<script>` tags.
```

Comment:
There’s a bit of repetition now that this has moved here. L9 & L13:

> You can add interactivity to your Astro components […] using standard HTML `<script>` tags.

> Astro components support adding client-side interactivity using standard HTML `<script>` tags.

I’ll also call out that this content used to be in astro-components as a kind of hint that this was possible, linking here. This means that a) now that it’s on this page, it’s repeats some stuff later on this page and b) we no longer call-out client-side scripts (or styles!) are possible anywhere in either the `astro-components` or the `astro-syntax` page, which doesn’t seem ideal.

Reply:
I believe I addressed this!